pkg/resourceinterpreter/customized/declarative/luavm/lua.go

```diff
@@ -197,6 +197,33 @@ func (vm *VM) setLib(l *lua.LState) error {
 			return err
 		}
 	}
+
+	// Set potentially unsafe basic functions to nil, prohibiting users from
+	// calling these functions in custom Lua scripts.
+	// For users, these functions are not needed for parsing resource templates,
+	// so disabling these functions has no impact on functionality.
```

Comment:
Shall we link to the documents that claim these functions are not safe?
Is it https://lua-users.org/wiki/SandBoxes?

Reply:
+1